src/content_sao.h

```diff
@@ -251,6 +258,13 @@ class PlayerSAO : public UnitSAO
 	void setWantedRange(const s16 range);
 	s16 getWantedRange() const { return m_wanted_range; }
 
+	bool isImmortal()
+	{
+		return itemgroup_get(getArmorGroups(), "immortal") ||
+				!g_settings->getBool("enable_damage") ||
+				!g_settings->getBool("enable_pvp");
```

Comment:
players will no longer die from falling damage when PvP is disabled.
yeah, no

Reply:
Hmm, true. So PvP would have to be checked separately, I suppose. Good catch!